pkg/ksync/doctor/docker.go

```diff
@@ -95,3 +97,35 @@ func IsDockerStorageCompatible() error {
 
 	return nil
 }
+
+// IsDockerGraphMatching checks to see if the configured graph directory (the location of the graph driver storage directories) matches what's configured in radar
+func IsDockerGraphMatching() error {
+	nodes, err := cluster.NewService().NodeNames()
+	if err != nil {
+		return err
+	}
+
+	for _, node := range nodes {
```

Comment:
We need to create a helper for this pattern as it keeps coming up in the doctor methods. (future issue)